### ISSUE TITLE
Problem publishing all docs

### ISSUE THREAD

**Issue by arthurspa** [jacebrowning/doorstop#279]
I was following main page tutorial and got an error when executing:
`doorstop publish all ./public`

I'm using python 3.6.0 in Windows 7 and virtualenv.

```
(venv)
arthur.s@MODEL-ARTHURS MINGW32 /d/dev/temp/doorstop
$ doorstop publish all ./public
building tree...
loading documents...
publishing tree to 'D:\dev\temp\doorstop\public'...
ERROR: Problem parsing the template sidebar
Traceback (most recent call last):
  File "c:\users\arthur.s\appdata\local\programs\python\python36-32\Lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\users\arthur.s\appdata\local\programs\python\python36-32\Lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "D:\Dev\temp\rmtoo\venv\Scripts\doorstop.exe\__main__.py", line 9, in <module>
  File "d:\dev\temp\rmtoo\venv\lib\site-packages\doorstop\cli\main.py", line 104, in main
    success = function(args, os.getcwd(), parser.error)
  File "d:\dev\temp\rmtoo\venv\lib\site-packages\doorstop\cli\commands.py", line 504, in run_publish
    template=args.template, **kwargs)
  File "d:\dev\temp\rmtoo\venv\lib\site-packages\doorstop\core\publisher.py", line 84, in publish
    common.write_lines(lines, path2)
  File "d:\dev\temp\rmtoo\venv\lib\site-packages\doorstop\common.py", line 152, in write_lines
    for line in lines:
  File "d:\dev\temp\rmtoo\venv\lib\site-packages\doorstop\core\publisher.py", line 223, in publish_lines
    yield from gen(obj, **kwargs)
  File "d:\dev\temp\rmtoo\venv\lib\site-packages\doorstop\core\publisher.py", line 509, in _lines_html
    html = bottle_template(template, body=body, toc=toc_html, parent=obj.parent)
  File "d:\dev\temp\rmtoo\venv\scripts\bottle.py", line 3615, in template
    TEMPLATES[tplid] = adapter(name=tpl, lookup=lookup, **settings)
  File "d:\dev\temp\rmtoo\venv\scripts\bottle.py", line 3222, in __init__
    raise TemplateError('Template %s not found.' % repr(name))
bottle.TemplateError: Template 'sidebar' not found.
```

**Comment by jacebrowning** [comment]
@jmacarthur Any idea what's going on here?

**Comment by jmacarthur** [comment]
No idea right now, but it does look likely to be related to the changes I put in. Windows isn't my usual development environment - I have a Windows 7 machine, but I haven't got Doorstop to work from source yet. The pip-installed version of Doorstop seems to work correctly. I should be able to spend some more time looking at it this weekend.

**Comment by olegmentat** [comment]
Just found this project today and ran into the same problem (and I'm new to Python coming from C++). Seemed to be that setup.py is trying to include *.tpl files from "views" project:
`    package_data={'doorstop.core': ['files/*.html', 'files/*.css', 'files/assets/doorstop/*'],
                  'views': ['*.tpl']},`
But "views" is not getting picked up as a project, so those views templates are not copied over to the install path.

I fixed it on my box by adding **__init__.py** file in folder **doorstop\views** next to the *.tpl files. Seemed to work, but I'm not sure if that's the right approach in Python.

**Comment by jacebrowning** [comment]
That sounds like the issue https://github.com/jacebrowning/doorstop/pull/272 was attempting to fix. 

You could also `pip install doorstop==1.2.1` as a workaround until we have time to investigate this.

**Comment by jmacarthur** [comment]
I've raised #280 which should fix this, pending review. Thanks @arthurspa for reporting this and @olegmentat for the fix - I've done it a different way in the merge request, but adding a blank `__init__.py` to views will work if you need to work around this temporarily.

**Comment by jacebrowning** [comment]
This should be fixed with `pip install doorstop==1.3.1`.

**Comment by olegmentat** [comment]
Just wanted to confirm that the views folder is included and the publishing works on Windows 10, both installing as `pip install doorstop` and directly from source with `python setup.py install`
